### PR TITLE
project-movies-PriscilaAlfaro

### PR DESCRIPTION
**Pull request by PriscilaAlfaro**
Link to my deployed project: https://project-movies-priscilaalfaro.netlify.app/

My reflections on how this week's project turned out: Very nice practice to learn about React-router.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -19,28 +23,29 @@ export const App = () => {
...
-        console.log('Error in Fetch:' + error.message);
-      });
-  }
-
   useEffect(fetchMovies, [])
 
+  // Study Notes: 
+  // Just for academic example I will keep one Route with render, and another one with open and closing tags
+  // Option when no passing props <Route path="/" exact component={Home} /> 
+  // 'exact' is just necesary when overlaping characteres like '/', do not overuse!
```
Comment (Svempolin) [inline code/src/App.js:31]: Really nice that you comment.

File: code/src/App.js [hunk 2]
```diff
@@ -19,28 +23,29 @@ export const App = () => {
...
     <BrowserRouter>
       <Nav />
       <main className="main-container">
         <Switch>
-          <Route path="/" exact>
-            <Home movies={movies} fetchMovieDetails={fetchMovieDetails} />
-          </Route>
-          <Route path="/movie-details/:id" exact>
-            <MovieDetails movies={movies} movieDetails={movieDetails} />
+          <Route path="/" exact render={() => <Home movies={movies} />} />
```
Comment (Svempolin) [inline code/src/App.js:38]: Didn´t really understand the "<Home" but I guess is it make the page go "home"

File: code/src/Pages/NotFound/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    const onPreviousPageRedirect = () => {
+        history.goBack();
+    };
+    return (
+        <section className="notFound-container">
+            <h1 className="notFound-title">Sorry, We can not find that movie in our data base</h1>
+            <button className="notFound-button" onClick={onHomePageRedirect}>Return to home</button>
+            <button className="notFound-button" onClick={onPreviousPageRedirect}>Go back</button>
+            <img className="notFound-gif" src="https://media.giphy.com/media/A9EcBzd6t8DZe/giphy.gif" alt="Not Found" />
```
Comment (Svempolin) [inline code/src/Pages/NotFound/index.js:20]: This giphy was really nice!

**Review by Svempolin — COMMENTED** [review]
We think that you have done a great job and your page looks very nice! We think you have fulfilled the requirements for blue and red level, well done! Here are some comments which could maybe improve your code.
In your Readme you have first copied the brief, it could be better to rewrite it so that its “You” that describes the project. 
Your indentations look very nice, it's easy to read your code from that point of vue!

File: code/src/Components/MovieCard/movieCard.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: rgba(0,0,0,.75);
+    display: none;
+    padding: 20px;
+    z-index: 3;
+}
+
+.movieCard-image{
```
Comment (louise-becker) [inline code/src/Components/MovieCard/movieCard.css:22]: When reviewing we saw that the second movie from the rights img on the first page is smaller than others/the blue background is not in line with other movies. So you could maybe set a fixed size to the pictures or adjust the height.

**Review by louise-becker — COMMENTED** [review]
Same comments as @Svempolin - well done!

**Pull request closed without merging**